app/main/presenters/search_summary.py

```diff
@@ -31,7 +25,16 @@ def write_list_as_sentence(input_list, final_conjunction):
             formatted_conjunction = " {} ".format(final_conjunction)
             return formatted_conjunction.join([u', '.join(start), end])
 
-    def __init__(self, results_total, request_args, filter_groups, lots_by_slug):
+    def __init__(self, results_total, request_args, filter_groups, lots_by_slug, include_markup=True):
+        self.include_markup = include_markup
+
+        self.COUNT_PRE_TAG = '<span class="search-summary-count">'
```

Comment:
Could these go back to being class-level? Makes a bit more sense for constants really.